ui/component/channelThumbnail/index.js

```diff
@@ -1,14 +1,24 @@
 import { connect } from 'react-redux';
-import { selectThumbnailForUri, selectClaimForUri, selectIsUriResolving } from 'redux/selectors/claims';
+import {
+  selectThumbnailForUri,
+  selectClaimForUri,
+  selectIsUriResolving,
+  selectClaimsByUri,
+} from 'redux/selectors/claims';
 import { doResolveUri } from 'redux/actions/claims';
+import { selectOdyseeMembershipByClaimId } from 'redux/selectors/user';
+import { doFetchUserMemberships } from 'redux/actions/user';
 import ChannelThumbnail from './view';
 
 const select = (state, props) => ({
   thumbnail: selectThumbnailForUri(state, props.uri),
   claim: selectClaimForUri(state, props.uri),
   isResolving: selectIsUriResolving(state, props.uri),
+  selectOdyseeMembershipByClaimId: selectOdyseeMembershipByClaimId(state, props.uri),
```

Comment:
Bad prop name.   There are many instances of this.
- [ ] Should be something like this, since "select" is a verb/function.
     - `odyseeMembershipByClaimId: selectOdyseeMembershipByClaimId(state, props.uri),`
- [ ] Code says "by claim id", but we are passing in URI.